static/js/components/CardGrid.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import CardComponent from "./CardComponent"
+
+require('../../css/CardComponent.scss');
```

Comment:
Fine with it for now but lets make an issue to revisit this part of the scss approach, not a fan